ql/src/test/org/apache/hadoop/hive/ql/io/orc/TestOrcFile.java

```diff
@@ -1234,7 +1234,7 @@ public void emptyFile() throws Exception {
     assertEquals(false, reader.rows().hasNext());
     assertEquals(CompressionKind.NONE, reader.getCompression());
     assertEquals(0, reader.getNumberOfRows());
-    assertEquals(0, reader.getCompressionSize());
+    assertEquals(262144, reader.getCompressionSize());
```

Comment:
this looks a bit weird...I mean any non-zero to non-zero change is okay as file size changes, but 0 -> 262144 is strange, do we know the root cause? was the assertion of =0 made intentionally?

Reply:
I agree it looks weird. The reason is this:
ORC-1357: Handle missing compression block size (#1385)
https://issues.apache.org/jira/browse/ORC-1357

Reply:
yay, very cool, nice catch!
so we had previously made a completely invalid assertion without a code comment (without being aware that we're asserting based on an invalid behavior)